Protacon.NetCore.WebApi.TestUtil/Extensions/CallResponseExtensions.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.Net;
+using System.Threading;
+
+namespace Protacon.NetCore.WebApi.TestUtil.Extensions
+{
+    public static class CallResponseExtensions
+    {
+        public static CallResponse WaitForStatusCode(this CallResponse response, HttpStatusCode statusCode, TimeSpan timeout)
+        {
+            DateTime endAt = DateTime.Now.Add(timeout);
```

Comment:
Not sure but this and the comparison below may cause a case when error/bug when this timeout happens during daylight saving time.

At least the `DateTime.Add()` doesn't seem to move the `DateTime` to expected value, and I think the comparison below with `DateTime.Now` might cause the exception to be thrown prematurely if clock is moved forward, if `DateTime.Now` returns the actual time.

```
var start = new DateTime(2018, 3, 25, 2, 59, 00, DateTimeKind.Local);
var end = start.Add(TimeSpan.FromMinutes(30));
```
End was 25.3.2018 03.29.00, instead of 04.29.00